tasks/configure.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+- name: Prometheus Node Exporter service start
+  become: yes
+  service:
+    name: prometheus-node-exporter
+    state: started
```

Comment:
This needs the when clause to stop it from firing when prometheus_node_exporter.start_on_boot is set to false: 

```when: prometheus_node_exporter.start_on_boot```

Reply:
Ah I see what's making the tests fail, give me a shout when you're free.